<launch>
    <node name="optimize_server" pkg="optimize_path" type="server_optimize.py"/>
</launch>